sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -329,6 +329,7 @@ private HttpPipeline createHttpPipeline() {
         return new HttpPipelineBuilder()
             .policies(policies.toArray(new HttpPipelinePolicy[0]))
             .httpClient(httpClient)
+                .tracer(new NoopTracer())
```

Comment:
I think I'd rename this PR to "Fix GlobalOpenTelemetry usage", to make it clearer that it affects customers too